src/libutil/error.hh

```diff
@@ -178,28 +178,55 @@ MakeError(Error, BaseError);
 MakeError(UsageError, Error);
 MakeError(UnimplementedError, Error);
 
-class SysError : public Error
+/**
+ * To use in catch-blocks.
+ */
+MakeError(SystemError, Error);
+
+/**
+ * POSIX system error, created using `errno`, `strerror` friends.
+ *
+ * Throw this, but do not catch this! Catch `SystemError` instead. This
```

Comment:
Catching `SysError` is fine if you need to specifically test `errNo`.

Reply:
Adjusted docs to describe this